src/main/java/com/sublinks/sublinksapi/post/services/PostLikeService.java

```diff
@@ -55,13 +57,8 @@ private void updateOrCreatePostLike(final Post post, final Person person, final
 
   private void createPostLike(final Post post, final Person person, final int score) {
 
-    final PostLike postLike = PostLike.builder()
-        .post(post)
-        .person(person)
-        .isUpVote(score == 1)
-        .isDownVote(score == -1)
-        .score(score)
-        .build();
+    final PostLike postLike = PostLike.builder().post(post).person(person).isUpVote(score == 1)
+        .isDownVote(score == -1).score(score).build();
```

Comment:
Why does your IDE collapse these down? I think it's more difficult to read this way.

Reply:
Fixed it, there was something wrong in the editorconfig file